### PR TITLE
Change code language to english

### PR DESCRIPTION
**Pull request by Seagullible**

### CODE REVIEW

**Review by abc013 — APPROVED** [review]
Seems ok to me. @abcdefg30 what's your meaning?

File: settings.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-H�he: 16
-Breite: 16
-Spieler: 4
+Height: 16
+Width: 16
+Player: 4
```
Comment (abcdefg30) [inline settings.txt:3]: One final minor nit: `Player` -> `Players`.
Sorry for not picking this up before.

**Comment by abc013** [comment]
Any progress @Seagullible?

**Comment by abcdefg30** [comment]
No progress since quite some time, and resurrecting this proofed to be more difficult than expected (due to recent changes in the code). Superseded by #15. Sorry.

**Pull request closed without merging**